tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -2349,6 +2354,17 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template name="ODM.oidReference">
+    <xsl:param name="annos" />
+    <xsl:variable name="sont" select="$annos/edm:Annotation[not(@Qualifier) and (@Term=$odmOidReference or @Term=$odmOidReferenceAliased)]/edm:Record" />

```

Comment:
Better name this variable `anno`, as in the preceding template. It is definitely _not_ a SONT.

Reply:
Done as requested.